src/styles/ThemeStyles.ts

```diff
@@ -36,6 +39,10 @@ export const theme = createMuiTheme({
       fontSize: '12px',
       fontWeight: 500,
     },
+    body1: {
```

Comment:
I think main already has a `body1` (changed h4 to body1) with a fontSize of 12 -- might need to change this to `body2` when merging